### PR TITLE
feat(general): allow listing and confirming users manually. Fixes MA-77

### PR DESCRIPTION
**Pull request by serge1peshcoff**
frontend for https://github.com/AEGEE/core/pull/152

### CODE REVIEW

File: src/components/layout/Sidebar.vue [hunk 1]
```diff
@@ -5,8 +5,8 @@
         {{ category.categoryName }}
       </p>
       <ul class="menu-list">
-        <li v-for="item in category.components" exact v-bind:key="item.name">
-          <router-link :to="{ name: item.name, params: item.params }" v-if="item.name && !item.link" :aria-expanded="isExpanded(item) ? 'true' : 'false'" @click.native="toggle(item.index, category.index, item)">
+        <li v-for="item in category.components" v-bind:key="item.name">
```
Comment (WikiRik) [inline src/components/layout/Sidebar.vue:8]: Why did you change this?

Comment (serge1peshcoff) [reply src/components/layout/Sidebar.vue:8]: This is wrong right now, and it results on Members menu entry being highlighted when you are on List unconfirmed users page (probably because they have same prefix /members)

File: src/views/core/members/ListUnconfirmed.vue [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+              <router-link :to="{ name: 'oms.members.view', params: { id: props.row.username || props.row.id } }">
+                {{ props.row.first_name }} {{ props.row.last_name }}
+              </router-link>
+            </b-table-column>
+
+            <b-table-column field="email" label="Email">
+              {{ props.row.email }}
+            </b-table-column>
+
+            <b-table-column field="date_of_birth" label="Birthday" sortable width="150">
```
Comment (WikiRik) [inline src/views/core/members/ListUnconfirmed.vue:39]: I don't think we need the birthday, address and about me fields here

**Review by WikiRik — COMMENTED** [review]
Check my comments on the ListUnconfirmed.vue file

File: src/components/layout/Sidebar.vue (hunk 1 above)
Comment (WikiRik) [reply src/components/layout/Sidebar.vue:8]: Ah yes, sounds good!

**Review by WikiRik — APPROVED** [review]

File: src/views/core/members/ListUnconfirmed.vue (hunk 2 above)
Comment (serge1peshcoff) [reply src/views/core/members/ListUnconfirmed.vue]: makes sense, deleted these fields
